content/en/xtc/integrations/510-slack.md

```diff
@@ -94,6 +94,10 @@ If a complete configuration (i.e. integration data and target channels) is avail
 
 Once configured XTC will send notifications for the following events:
 
+{{% warning notitle %}}
+Due to limitations in Slack's text renderer certain control characters will be filtered from the input values when displaying them in messages. If any of your organization's, project's, load test's or report's names containes any of the following characters, these may be removed: `_`, `*`, `` ` `` or `~`.
```

Comment:
Please adjust spelling: Due to limitations of Slack's text renderer, certain control characters (`_`, `*`, `` ` ``, `~`) have to be removed. This applies to organization, project, load test, and report names. XTC does not limit your freedom here when setting data up, but this won't make it into the Slack communication.

Reply:
Done